.shellcheckrc

```diff
@@ -32,3 +32,15 @@ disable=SC3013
 # https://github.com/koalaman/shellcheck/wiki/SC3045
 # ... but dash supports it
 disable=SC3045
+
+# SC2086: Double quote to prevent globbing and word splitting.
+# https://github.com/koalaman/shellcheck/wiki/SC2086
+disable=SC2086
```

Comment:
I do not see why would you want not to quote `"$@"`. I'd rather solve this on case-by-case basis, as globally disabling this might lead to possibly faulty code.